docs/input_file_overview.md

```diff
@@ -36,10 +36,10 @@ Body: The body of the VCF file contains the actual variant data, with each row r
-| Sample            | Projects consist of analyses that are run on samples. We accept sample information in the form of BioSample, ENA or EGA accession(s). We also accept BioSamples sampleset accessions. If your samples are not yet accessioned, and are therefore novel, please use the "Novel sample(s)" sections of the Sample(s) worksheet to have them registered at BioSample |
-| Analysis          | For EVA, each analysis is one vcf file, plus an unlimited number of ancillary files. This sheet allows EVA to link vcf files to a project and to other EVA analyses. Additionally, this worksheet contains experimental meta-data detailing the methodology of each analysis. Important to note; one project can have multiple associated analyses                |
-| Files             | Filenames and associated checking data associated with this EVA submission should be entered into this worksheet. Each file should be linked to exactly one analysis.                                                                                                                                                                                             |
+| WORKSHEET         | EXPLANATION                                                                                                                                                                                                                                                                                                                                                                                                                |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Submitter Details | This sheet captures the details of the submitter                                                                                                                                                                                                                                                                                                                                                                           |
+| Project           | The objective of this sheet is to gather general information about the Project. If you are submitting to an existing project, you can skip the other details and just provide the project accession and analyses will be added to that. In case of a new project, please provide the relevant details including submitter, submitting centre, collaborators, project title, description and publications and publications. |
```

Comment:
```suggestion
| Project           | The objective of this sheet is to gather general information about the Project. If you are submitting to an existing project, you can skip the other details and just provide the project accession and analyses will be added to that. In case of a new project, please provide the relevant details including submitter, submitting centre, collaborators, project title, description and publications. |
```